Makefile

```
@@ -275,13 +276,55 @@ e2e-dependencies:
+FAKE_SEARCH_HOST ?= http://127.0.0.1:$(FAKE_SEARCH_PORT)
+FAKE_SEARCH_PID_FILE ?= .fake_search.pid
 run-instrument:
 	kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
 	kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o jsonpath='{.data.tls\.crt}' | base64 -d > tls.crt
 	kubectl get secret ${SECRET_NAME} -n ${NAMESPACE} -o jsonpath='{.data.tls\.key}' | base64 -d > tls.key
 	go build -cover -o mtv_integrations_instrumented cmd/main.go
 	mkdir -p coverage_profiles
-	GOCOVERDIR=coverage_profiles nohup ./mtv_integrations_instrumented --webhook-cert-path=. >> test/e2e/e2e.log  2>&1 &
+	GOCOVERDIR=coverage_profiles nohup ./mtv_integrations_instrumented --webhook-cert-path=. $${THANOS_HOST:+--thanos-host=$${THANOS_HOST}} $${SEARCH_API_ENDPOINT:+--search-api-endpoint=$${SEARCH_API_ENDPOINT}} >> test/e2e/e2e.log  2>&1 &
+
+start-fake-thanos:
```

checkmake (0.3.2)

[warning] Target body for "start-fake-thanos" exceeds allowed length of 5 lines (10).

(maxbodylength)